### PR TITLE
Add large image check to hierarchical topic curations

### PR DESCRIPTION
**Pull request by nicola-d-bbc**
Resolves WSTEAMA-181

**Overall change:**
Adds a new 660 resolution to promo images, adds large image check for big desktop promos.

Note that unit tests don't seem to exist for the promo component but prioritised fixing bug than adding lots of tests.

**Code changes:**

- Add 660 resolution to imageResolution array
- Add large image boolean and passes as optional to image promo.

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project
- [ ] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/legacy/components/Promo/image.jsx [hunk 1]
```diff
@@ -55,7 +59,7 @@ const Image = props => {
     <Wrapper>
       <IMAGE
         {...rest}
-        src={src.replace('{width}', 240)}
+        src={src.replace('{width}', 660)}
```
Comment (eagerterrier) [inline src/app/legacy/components/Promo/image.jsx]: Think you want lowest to appear here by default.

File: src/app/components/Image/index.stories.tsx [hunk 2]
```diff
@@ -18,7 +18,7 @@ export const ResponsiveImage = () => (
     <Image
       alt="A penguin stands on an ice floe"
       src="https://ichef.bbci.co.uk/news/624/cpsprodpb/164AF/production/_110911319_antartica.jpg"
-      srcSet="https://ichef.bbci.co.uk/news/240/cpsprodpb/164AF/production/_110911319_antartica.jpg 240w, https://ichef.bbci.co.uk/news/624/cpsprodpb/164AF/production/_110911319_antartica.jpg 624w"
+      srcSet="https://ichef.bbci.co.uk/news/660/cpsprodpb/164AF/production/_110911319_antartica.jpg 660w, https://ichef.bbci.co.uk/news/624/cpsprodpb/164AF/production/_110911319_antartica.jpg 624w"
```
Comment (eagerterrier) [inline src/app/components/Image/index.stories.tsx]: don't think you need this change

File: src/app/legacy/components/Promo/image.jsx (hunk 1 above)
Comment (nicola-d-bbc) [reply src/app/legacy/components/Promo/image.jsx]: Was just testing it out while it was in draft!

Comment (pvaliani) [reply src/app/legacy/components/Promo/image.jsx]: very eager @eagerterrier 👀

File: src/app/components/Image/index.stories.tsx (hunk 2 above)
Comment (nicola-d-bbc) [reply src/app/components/Image/index.stories.tsx]: Nope, was just testing it out :)

**Review by pvaliani — APPROVED** [review]
All good, think it just needs zee unit tests.

**Review by Louis-Matsika — APPROVED** [review]

File: src/app/components/Image/index.stories.tsx [hunk 3]
```diff
@@ -18,7 +18,7 @@ export const ResponsiveImage = () => (
     <Image
       alt="A penguin stands on an ice floe"
       src="https://ichef.bbci.co.uk/news/624/cpsprodpb/164AF/production/_110911319_antartica.jpg"
-      srcSet="https://ichef.bbci.co.uk/news/240/cpsprodpb/164AF/production/_110911319_antartica.jpg 240w, https://ichef.bbci.co.uk/news/624/cpsprodpb/164AF/production/_110911319_antartica.jpg 624w"
+      srcSet="https://ichef.bbci.co.uk/news/240/cpsprodpb/164AF/production/_110911319_antartica.jpg 660w, https://ichef.bbci.co.uk/news/624/cpsprodpb/164AF/production/_110911319_antartica.jpg 624w"
```
Comment (eagerterrier) [inline src/app/components/Image/index.stories.tsx]: this doesn't look right to me.

Isn't this saying when the image is 660px wide, we serve up an image 240px wide?

Comment (eagerterrier) [reply src/app/components/Image/index.stories.tsx]: tbh, the image srcset and sizes isn't my strongpoint, so I may be way off the mark

Comment (nicola-d-bbc) [reply src/app/components/Image/index.stories.tsx]: Just forgot to change this from some visual tests we were doing in draft.

**Review by amoore108 — APPROVED** [review]

**Comment by pvaliani** [comment]
@amoore108 @eagerterrier should a ticket be made for adding unit tests for this work separately or...?

**Comment by amoore108** [comment]
> @amoore108 @eagerterrier should a ticket be made for adding unit tests for this work separately or...?

I'd add them into here if there's new ones that need created.

**Comment by amoore108** [comment]
@pvaliani @nicola-d-bbc At some stage it may be worthwhile 'upgrading' the Promo component and moving into the main `components` folder and applying the component standards to it. We seem to be using it enough for new pages, so it makes sense to keep it updated.

**Comment by pvaliani** [comment]
> @pvaliani @nicola-d-bbc At some stage it may be worthwhile 'upgrading' the Promo component and moving into the main `components` folder and applying the component standards to it. We seem to be using it enough for new pages, so it makes sense to keep it updated.

I've created a ticket for this with reference WSTEAMA-206 :)

**Pull request merged**